apps/judicial-system/web/src/routes/Court/InvestigationCase/Confirmation/Confirmation.tsx

```diff
@@ -32,56 +32,91 @@ import {
 } from '@island.is/judicial-system-web/src/graphql/schema'
 import { useCase } from '@island.is/judicial-system-web/src/utils/hooks'
 
+import {
+  JudgeRequestRulingSignatureModal,
+  RegistrarRequestRulingSignatureModal,
+  RulingModifiedModal,
+} from '../../components'
 import { confirmation as strings } from './Confirmation.strings'
 
-type VisibleModal = 'none' | 'rulingModifiedModal' | 'signingModal'
+type VisibleModal =
+  | 'none'
+  | 'rulingModifiedModal'
+  | 'judgeRequestRulingSignatureModal'
+  | 'registrarRequestRulingSignatureModal'
+  | 'signingModal'
 
-const Confirmation = () => {
+const Confirmation: React.FC<React.PropsWithChildren<unknown>> = () => {
```

Comment:
This component does not use children, so the `<React.PropsWithChildren<unknown>>` is unneeded.